main.go

```diff
@@ -44,14 +44,18 @@ type ExportProcessor func(warehouse.Warehouse, []string, *fullstory.Client, []fu
 // For existing export table fields that do not exist in the json record, an empty string is populated.
 func TransformExportJSONRecord(wh warehouse.Warehouse, tableColumns []string, rec map[string]interface{}) ([]string, error) {
 	var line []string
-	// Change all record keys to lower case. We do this because columns are case insensitive for most warehouse solutions.
-	rec = getRecordWithLowerCaseKeys(rec)
-
-	// Map of CustomVars
+	lowerRec := make(map[string]interface{})
 	customVarsMap := make(map[string]interface{})
+
+	// Do a single pass over the data record to:
+	// a) extract all custom variables
+	// b) change standard/non-custom field names to lowercase for case-insensitive column name matching
```

Comment:
You could argue here that we lose some readability by removing the `getRecordWithLowerCaseKeys` function and doing everything in one pass.  However, `TransformExportJSONRecord` is a function that will be called on EVERY record in the bundle. This is a case where I think a slight loss of readability is an ok trade-off for saving those iterations.